rpcs3/Emu/NP/rpcn_config.h

```diff
@@ -15,18 +16,27 @@ struct cfg_rpcn : cfg::node
 	void save() const;
 
 	std::string get_host() const;
+	std::string get_country() const;
 	std::string get_npid(); // not const because it can save if npid is requested and it has never been set
 	std::string get_password() const;
 	std::string get_token() const;
 	std::vector<std::pair<std::string, std::string>> get_hosts(); // saves default if no valid server in the list
 
 	void set_host(std::string_view host);
+	void set_country(std::string_view country);
 	void set_npid(std::string_view npid);
 	void set_password(std::string_view password);
 	void set_token(std::string_view token);
 	bool add_host(std::string_view description, std::string_view host);
 	bool del_host(std::string_view description, std::string_view host);
 
+	struct country_code
+	{
+		std::string name;
+		std::string ccode;
```

Comment:
Given the way you initialize those (especially after the array becomes `constexpr`), you should be fine having those as `string_view`.